tree/dataframe/inc/ROOT/RDF/RDefineBase.hxx

```diff
@@ -41,17 +43,14 @@ protected:
    std::vector<Long64_t> fLastCheckedEntry;
    RDFInternal::RBookedDefines fDefines;
    std::deque<bool> fIsInitialized; // because vector<bool> is not thread-safe
-   const std::map<std::string, std::vector<void *>> &fDSValuePtrs; // reference to RLoopManager's data member
-   ROOT::RDF::RDataSource *fDataSource; ///< non-owning ptr to the RDataSource, if any. Used to retrieve column readers.
+   RLoopManager *fLoopManager; // non-owning pointer to the RLoopManager
```

Comment:
All its uses (that I saw + checked) seem to suggest that a `const RLoopManager *` would do?

Reply:
`fLoopManager->Deregister(this)` is non-const